test/unittest_schemes.cxx

```diff
@@ -82,22 +73,9 @@ class Ut_Schemes_Test : public Fl_Group {
   }
   Ut_Schemes_Test(int X,int Y,int W,int H)
     : Fl_Group(X,Y,W,H) {
-    scheme_choice_ = new Fl_Choice(X+125,Y,140,25,"FLTK Scheme");
```

Comment:
Nice reduction of code here, and easier to maintain. Thanks!

Reply:
My goal is still to be able to add new schemes (or let users add new schemes) without patching the FLTK code in several (many!) places. I'm making good progress - step by step - and eventually this should work. And I also hate having to find all the places if we add new schemes or similar things. Maintainability is important!